sql: TestNextRowInterleaved fails with small FastIntSet cutoff

Before every release, I run `make test` with a change to FastIntSet that forces most usage into the "large set mode". This catches bugs where the structure isn't used correctly - typically unwanted aliasing due to missing `Copy()` calls.

To repro, apply this diff:
```
--- a/pkg/util/fast_int_set.go
+++ b/pkg/util/fast_int_set.go
@@ -39,7 +39,7 @@ func MakeFastIntSet(vals ...int) FastIntSet {
 
 // We store bits for values smaller than this cutoff.
 // Note: this can be set to a smaller value, e.g. for testing.
-const smallCutoff = 64
+const smallCutoff = 2
```

```
--- FAIL: TestNextRowInterleaved (9.46s)
    --- FAIL: TestNextRowInterleaved/parent1-parent2-child1-grandchild1-grandgrandchild1-child2-ggc1_unique_idx-parent3 (0.00s)
        fetcher_test.go:986: for table grandgrandchild1@ expected 5 non-NULL values, got 4
    --- FAIL: TestNextRowInterleaved/parent1-child1-grandchild1-grandgrandchild1-child2-ggc1_unique_idx-parent3 (0.00s)
        fetcher_test.go:986: for table grandgrandchild1@ expected 5 non-NULL values, got 4
    --- FAIL: TestNextRowInterleaved/parent2-child1-grandchild1-grandgrandchild1-child2-ggc1_unique_idx-parent3 (0.00s)
        fetcher_test.go:986: for table grandgrandchild1@ expected 5 non-NULL values, got 4
    --- FAIL: TestNextRowInterleaved/parent1-parent2-grandchild1-grandgrandchild1-child2-ggc1_unique_idx-parent3 (0.00s)
        fetcher_test.go:986: for table grandgrandchild1@ expected 5 non-NULL values, got 4
    --- FAIL: TestNextRowInterleaved/parent1-grandchild1-grandgrandchild1-child2-ggc1_unique_idx-parent3 (0.00s)
        fetcher_test.go:986: for table grandgrandchild1@ expected 5 non-NULL values, got 4
    --- FAIL: TestNextRowInterleaved/parent2-grandchild1-grandgrandchild1-child2-ggc1_unique_idx-parent3 (0.00s)
...
```

--- a/pkg/sql/row/fetcher_test.go
+++ b/pkg/sql/row/fetcher_test.go
@@ -849,6 +849,7 @@ func TestNextRowInterleaved(t *testing.T) {
 	ggc1idx.indexName = "ggc1_unique_idx"
 	ggc1idx.indexIdx = 1
 	// Last column v (idx 4) is not stored in this index.
+	ggc1idx.valNeededForCol = ggc1idx.valNeededForCol.Copy()
 	ggc1idx.valNeededForCol.Remove(4)
 	ggc1idx.nVals = 4
 

